### ISSUE TITLE
apm-server multiple ingress rules.

### ISSUE THREAD

**Issue by CyberHippo** [elastic/helm-charts#1047]
It would be nice for the apm-server `ingress.yaml` to support multiple host/path rules like logstash or elasticsearch does.
Maybe through changes similar to #994.
Thanks.

**Comment by jmlrt** [comment]
Hi @CyberHippo,
Thanks for reporting this request.
Would you be interested to provide a PR like #994 for it?

**Comment by CyberHippo** [comment]
Hi @jmlrt,
Yes, I'll take a look as soon as possible.

**Comment by rkennedy-tpl** [comment]
I was looking into APM+RUM and I think I need to create a public-facing ingress and expose PATHs to customer browsers.

I went into the apm-server source and found the specific paths used for RUM ingest => https://github.com/elastic/apm-server/blob/f4e0620ccaa090315d17869d0e5ac4bbdf66b5f7/beater/api/mux.go#L59-L67

so the ideal solution is flexible enough that I can allow only `/intake/v2/rum/events` and `/intake/v3/rum/events` on a public-facing ingress and block all other PATHs.

I may want `/assets/v1/sourcemaps` (possibly with different rules/restrictions) so my CICD service can upload sourcemaps. I'm not planning on using `/config/v1/rum/agents` for auto-config, but someone else might want it.

I've found the other elastic helm-chart ingresses plenty flexible for similar goals, so something similar to #994 should be sufficient for my needs too.
